AlphaWallet/Settings/Views/SaveCustomRpcView.swift

```diff
@@ -137,11 +141,20 @@ class SaveCustomRpcView: UIView {
     }
 
     func resetAllTextFieldStatus() {
-        let fields: [TextField] = [chainNameTextField, rpcEndPointTextField, chainIDTextField, symbolTextField, explorerEndpointTextField]
-        for field in fields {
+        for field in allTextFields {
             field.status = .none
         }
     }
+
+    func focusOnTextField(textField: TextField) {
+        let offset = textField.frame.maxY - scrollView.bounds.height
+        guard
```

Comment:
Can you make this 1 (long) line? With the idea to let editors soft-wrap so it's still readable and we don't have to fiddle with indentation.